eth/signer.go

```diff
@@ -100,24 +100,31 @@ func (s *Signer) Address() common.Address {
 // SHOULD be hashed first to avoid chosen-plaintext attacks. Prefer
 // Signer.Sign().
 func (s *Signer) RawSign(buf []byte) ([]byte, error) {
-	return crypto.Sign(buf, s.key)
+	sig, _, err := s.sign(buf, signOpts{
+		raw:       true,
+		compact:   false,
```

Comment:
I was not entirely sure about that either. I opted for `compact: false` in the case of raw signatures, because people might expect to get a standard ECDSA signature.

That being sad, I could also get fully behind the idea of saying that we only use compact signatures in ethier.